### DISCUSSION TITLE
[Hiring / Job Opportunity]

### DISCUSSION THREAD

**Discussion by devinschumacher** [arthelokyo/astrowind#389] [General]
Looking to connect with someone who's worked on/built the Astrowind theme to do some customizations on our astro.build sites (that use astrowind). 

If that's you, please shoot me a message somewhere on social!

- https://linkedin.com/in/devinschumacher
- https://instagram.com/dvnschmchr
- https://www.facebook.com/mdevinschumacher

cheers

**Comment by prototypa** [comment]
Hi @devinschumacher 

We'll be in touch. Thank you.
